libs/net/ches/ChesService.cs

```diff
@@ -0,0 +1,310 @@
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using TNO.Ches.Configuration;
+using TNO.Ches.Models;
+using TNO.Core.Exceptions;
+using TNO.Core.Extensions;
+using TNO.Core.Http;
+using TNO.Core.Http.Models;
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Net.Http;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace TNO.Ches
+{
+    /// <summary>
+    /// ChesService class, provides a service for integration with Ches API services.
+    /// </summary>
+    public class ChesService : IChesService
+    {
+        #region Variables
+        private readonly ClaimsPrincipal _user;
+        private readonly JwtSecurityTokenHandler _tokenHandler;
```

Comment:
Does not appear to be required.  Linting should have picked this up and said unused variable.

Reply:
Ooops thought I got rid of that